test/Index/roles.swift

```diff
@@ -74,6 +74,10 @@ func aCalledFunction(a: Int, b: inout Int) {
   // CHECK-NEXT: RelCont | function/Swift | aCalledFunction(a:b:) | s:14swift_ide_test15aCalledFunctionySi1a_Siz1btF
   // CHECK: [[@LINE-3]]:7 | param/Swift | a | s:{{.*}} | Ref,Read,RelCont | rel: 1
   // CHECK-NEXT: RelCont | function/Swift | aCalledFunction(a:b:) | s:14swift_ide_test15aCalledFunctionySi1a_Siz1btF
+
+  _ = { ignored in ignored + 1}
```

Comment:
Jump-to-def won't use the indexer in this case. It has an AST from working out what your cursor is over, so for locals it can return the definition's location with that, and never calls out to the indexer.